src/language/en-US.json

```diff
@@ -326,7 +326,6 @@
     "loading": "Loading...",
     "locked_mosaic": "locked mosaic",
     "login": "Login",
-    "low_fee_warning_message": "Transaction with a less than recommended fee might not get accepted and eventually",
```

Comment:
This line shouldn't be removed. I think you had some merge conflicts were not solved properly.